### PR TITLE
First shrinkwrap

### PR DESCRIPTION
**Pull request by gilmoskowitz**
Let's review together before pulling and make sure the `.travis.yml` changes address CI failures from #266. I'm unsure of the ramifications of adding the `xtuple-server-*` dependencies.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -9,6 +9,8 @@ env:
 
 before_install:
   - wget https://raw.githubusercontent.com/xtuple/xtuple-server-core/master/bootstrap.sh -qO- | sudo bash
+  - node --version
+  - n 0.11.13
```
Comment (tjwebb) [inline .travis.yml]: This will hide a potential failure of bootstrap.sh to install and declare the correct version of node (0.11.13) on its own.

**Comment by shackbarth** [comment]
@gilmoskowitz Merge conflict

**Comment by gilmoskowitz** [comment]
@shackbarth - merge conflict resolved & ci passed

**Comment by shackbarth** [comment]
@gilmoskowitz awesome. want to `npm publish` now?

**Pull request merged**